src/components/WriteButton/WriteButton.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import WriteButton from "./WriteButton";
+
+describe("WriteButton", () => {
+  test("renders a button", () => {
```

Comment:
Can you check the verbs? I don't see a standard being applied. Something you  call `renders` other times you call `execute`.

Reply:
Didn't you mean the use `renders` and `find`, because in my opinion `execute` was properly used, since we are trying to test if `onClick` was executed only once. Maybe I'm missing something?

Reply:
What I meant was:
```jsx
it("executes....", () => {})
it("finds...", () => {})
it("renders", () => {})
```

Reply:
Oh, gotcha! Indeed it feels weird that lack of pattern using plural